help/de/Help_ch_GUIMenus.xml

```diff
@@ -463,12 +475,12 @@
 
                 <entry nameend="c3" namest="c2">
                   <para>Speichert die aktuell geöffnete Datei unter einem anderen Namen.
+                    <footnote>
+                      <para>Bis zu &app; Version 2.6.x sollten Sie aufpassen, dass Sie Ihre Datendatei NICHT in
```

Comment:
Version &vers-last-2;

Reply:
Du könntest die Variable theoretisch auch auf 2.7.8, setzen, aber ich denke, wir sollten die Entwickler-Versionen in der offiziellen Doku normalerweise nicht berücksichtigen. Die sind ja nicht für normale Benutzer.
Und du müßtest für jedes Merkmal recherchieren, in welcher es genau geändert wurde.

Reply:
Sobald du ein nennenswertes zwischen 3 und 4 geändertes Merkmal hast, kannst du &vers-last-3; definieren.